test/manual/resource_management.test.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+
+import { AbstractCursor, ChangeStream, ClientSession, MongoClient } from '../mongodb';
+
+describe('Explicit Resource Management Tests', function () {
```

Comment:
The explicit resource management spec is more than just the `asyncDispose` symbol; considering the scope of these tests, I think it would be more accurate to say that this test file is for `'Symbol.asyncDispose [functionality in the driver]'`.